packages/core/src/waitFor.ts

```diff
@@ -6,7 +6,7 @@ interface WaitForOptions {
    * How long to wait before rejecting, if no emitted
    * state satisfies the predicate.
    *
-   * @default 10_000 (10 seconds)
+   * @defaultValue 10_000 (10 seconds)
```

Comment:
I'm quite surprised that this one matters - they are supposed to be synonyms in JSDoc 🤷

Reply:
It's worth noting that they display slightly differently in VS Code but the difference is kinda negligible and I'm not even sure which one is "better"/more expected in this context